api/python/provisioner/commands/validator/config_sections.py

```diff
@@ -120,3 +122,40 @@ def _validate_node_count(self, num_of_nodes, parsed):  # noqa: C901
         except Exception as exc:
             raise ValueError(f"Config Failed to apply: {str(exc)}")
 
+    def _validate_config_interfaces(self,config) :
+        """
+        Validate Interfaced passed to config.ini.
+        """
+        for section in config.sections() :
+            if ("srvnode_default" in section) :
```

Comment:
I think we can also provide interfaces in srvnode-1,2,3 section in config.ini in case of different interfaces on each node.

Reply:
+1 
Even though these are default section params, would be better to keep it generic to any `srvnode-`